.nvmrc

```diff
@@ -1 +1 @@
-10
+16.20.2
```

Comment:
After upgrading some packages, running with node 16 seems to be stable. Probably best to run with a more recent version and match version the datastax-docs-site repo uses